en.html

```diff
@@ -29,8 +29,8 @@
    <div class="hero-body">
        <div class="container has-text-centered">
            <h1 class="title">GDG Golang Kazan</h1>
-           <h2 class="subtitle"><a class="hero-link" href="https://golang.org/" target="_blank">Go</a> users group in Kazan.</h2>
-            <figure class="image"><img id="gopher" src="GolangKazan.jpg" class="is-rounded" alt="Логотип казанского сообщества: Гофер верхом на драконе Зиланте - символе г. Казань"></figure>
+           <h2 class="subtitle">A Kazan-based community for the <a class="hero-link" href="https://golang.org/" target="_blank">Go programming language</a> enthusiasts.</h2>

```

Comment:
I was trying to mimic other "communities", they are called "user groups" in the wild.
And "Go community" is a sum of all such user groups.

Example: https://www.meetup.com/golang-users-berlin/
> We are a group of Golang users.